build.sbt

```diff
@@ -20,40 +22,32 @@ lazy val shapelessVersion    = "2.3.2"
 lazy val slf4jVersion        = "1.7.25"
 lazy val tucoVersion         = "0.3.0-M5"
 
-enablePlugins(GitVersioning)
+// our version is determined by the current git state (see project/Version.scala)
+def imageManifest = ImageManifest.current("postgres:9.6.0").unsafeRunSync
 
-git.uncommittedSignifier in ThisBuild := Some("UNCOMMITTED")
+version in ThisBuild := imageManifest.formatVersion
 
 // check for library updates whenever the project is [re]load
-onLoad in Global := { s => "dependencyUpdates" :: s }
+onLoad in Global := { s => 
+  if (sys.props.contains("gem.skipDependencyUpdates")) s
+  else "dependencyUpdates" :: s 
+}
```

Comment:
In Jenkins I'm passing `-Dgem.skipDependencyUpdates` to speed things up. We should do this in Travis as well I guess.

Reply:
Do we really want to have 2 CIs setup? Could we just use travis and let github check the state from these somehow?

Reply:
We're not really doing CI on Jenkins … we're just packaging and deploying. We could move CI to Jenkins and have it poke GitHub but I haven't looked into this.

Reply:
Ok, then don't worry. I checked this a while ago and github can ping a local jenkins but we need to open the firewall which is a hassle